app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -150,6 +155,20 @@ class StatePlayerRecyclerViewAssembler private constructor(
 
   val isCorrectAnswer = ObservableField<Boolean>(false)
 
+  fun saveState(bundle: Bundle) {
+    bundle.putInt(KEY_TRACKED_WRONG_ANSWER_COUNT, hintHandler.trackedWrongAnswerCount)
+    // bundle.putSerializable(KEY_PREVIOUS_HELP_INDEX, hintHandler.previousHelpIndex as Serializable)
```

Comment:
@BenHenning here I'm unable to add previousHelpIndex to the bundle as it is a type of `HintIndex` how can I solve this

Reply:
Could you use the utility introduced in #2561 for this purpose? I suspect it may actually be easier just to create a new proto that contains the data the hint handler needs to store & only put that single proto in the bundle versus having multiple keys to manage.